dotnet-rpm/PackagingRunner.cs

```diff
@@ -44,7 +43,7 @@ public PackagingRunner(string outputName, string msbuildTarget, string commandNa
 
         public int Run(string[] args)
         {
-            CommandLineApplication commandLineApplication = new CommandLineApplication(throwOnUnexpectedArg: true);
+            CommandLineApplication commandLineApplication = new CommandLineApplication();
```

Comment:
Why is this change needed in the context of this PR?

Reply:
For `McMaster.Extensions.CommandLineUtils`, the ambition is to replace that dependency by [`System.CommandLine`](https://github.com/dotnet/command-line-api). I'd rather not update it if it's not required (e.g. to fix a bug); I've been bitten by breaking changes in the command line parsing packages in the past :-)

Reply:
Is this simpler now >netCore 2.1 is eol ?